src/components/shapediver/exports/ExportButtonComponent.tsx

```diff
@@ -28,7 +28,9 @@ export default function ExportButtonComponent(props: PropsExport) {
 				response.content[0] &&
 				response.content[0].href
 			) {
-				await fetchFileWithToken(response.content[0].href, `${response.filename}.${response.content[0].format}`);
+				const url = response.content[0].href;
+				const res = await actions.fetch(url);
```

Comment:
@snabela I don't quite get why there are two different fetch calls after another, could you explain that?

Reply:
@MajorMeerkatThe3rd I added a commit which explains this: https://github.com/shapediver/AppBuilderSdk/pull/77/commits/ea4479cb006e956d972ab52a3ca3721e44b22c6b
The `actions.fetch` function has knowledge of the token that was used to create the session. It returns a Response object, which is then used by `fetchFileWithToken` (reused and adapted from the platform frontend).

Reply:
@snabela all clear thanks